### ISSUE TITLE
Django 5 admin layout issue

### ISSUE THREAD

**Issue by offbyone** [dmptrluke/django-markdownfield#37]
This may apply to older versions, too, but it definitely applies here:

![Image](https://github.com/user-attachments/assets/9a545dac-d23e-4f90-ac2b-bfb1bdc7a9fa)

The markdown edit widget is really narrow and tall; it would be nice to have it below the name of the field, instead of beside it.

**Comment by benjaoming** [comment]
I only use this in LTS projects, so don't have access to quickly verify if this problem exists in Django 5... but it looks like static files are missing. Have you added `markdownfield` to `INSTALLED_APPS` and run `collectstatic` (if necessary) ?

**Comment by offbyone** [comment]
Yup, I've done both; it's definitely behaving a bit oddly. The `xxxxxxxx` at the top there is also not actually in the data.

**Comment by dmptrluke** [comment]
I'm currently working on updates for Django 4.2, but I'll have a look at this one after

**Comment by categulario** [comment]
I can attest that this happens in django 5.1 as well as 5.2 LTS. I can put some time in proposing a fix...

**Comment by benjaoming** [comment]
@categulario that sounds great! I can merge/review/release 👍

**Comment by dmptrluke** [comment]
I've done testing in the latest version, and this seems to be resolved. Please feel free to reopen after 0.12 releases if it is not!
